cxplat/inc/cxplat.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+#pragma once
+
+#include "cxplat/fault_injection.h"
+#include "cxplat/memory.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    int // 0 on success, errno value on failure.
```

Comment:
Let's define `CXPLAT_STATUS`, `CXPLAT_FAILED(X)` and `CXPLAT_SUCCEEDED(X)`, I assume in `common.h` and then use that instead of `int`.

Reply:
added CXPLAT_SUCCEEDED